lib/net.js

```diff
@@ -98,6 +98,10 @@ const {
 
 function noop() {}
 
+function getFlags(ipv6Only = false) {
```

Comment:
Unused default argument.

Reply:
The default value of `ipv6Only` would be actual `undefined` rather than `false` which is how its document described. This is the only reason I set the default argument.